MagneticField/GeomBuilder/plugins/dd4hep/DD4hep_VolumeBasedMagneticFieldESProducerFromDB.cc

```diff
@@ -74,12 +74,14 @@ namespace magneticfield {
 
     edm::ESGetToken<FileBlob, MFGeometryFileRcd> mayConsumeBlobToken_;
     const bool debug_;
+    const bool validate_;
```

Comment:
This file reads from the DB. We can assume the DB geometry has been checked, so it shouldn't need the validation flag. There is no need to modify this file.